pyproject.toml

```diff
@@ -71,6 +71,7 @@ dev = [
     "types-tabulate>=0.9.0.20241207,<0.10",
     "nominal-api-protos>=0.708.0",
     "coverage[toml]>=7.0.0",
+    "truststore>=0.10.4; python_version >= '3.10'",
```

Comment:
`truststore` is being imported in the main script, but isn't being installed for Python 3.9.

For consistency, easiest to drop Python 3.9 support (it's EOL October). If that's not an option, then either: (a) install it on all versions or (b) put a guard around the import and context configuration.